.gitignore

```diff
@@ -23,3 +23,4 @@ pip-wheel-metadata
 docs/docstree
 .docker
 molecule/**
+semaphore.lock
```

Comment:
Why not put this under `.tox`? Or even use a proper pytest location?

Reply:
done